docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+    - Validation method: Leave the default selection "DNS validation - recommended" selected.
+    - Key algorithm: Leave the default selection "RSA 2048" selected.
+![Enter certificate details page](../../../.gitbook/assets/aws-enter-cert-details-page.png)
+
+6. You should be redirected to the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) and there should be a banner at the top of the page that says "Successfully requested certificate with ID ...". Click "View certificate". This will take you to the certificate details page.
+![View certificate banner](../../../.gitbook/assets/aws-view-cert-banner.png)
+
+7. In the "Domains" section of the certificate details page, there should be a single row with `CNAME` listed under the "Type" column. The values in the "CNAME name" and "CNAME value" columns are used to validate that we control the domain (`civiform.dev`) requested by the certificate. Copy both the "CNAME name" and "CNAME value" column values somewhere for yourself temporarily, you will need them in the following steps.
+![Certificate details page](../../../.gitbook/assets/aws-copy-cname-values.png)
+
+8. In a new tab, open the [civiform.dev DNS configuration page](https://domains.google.com/registrar/civiform.dev/dns) and click "Manage custom records".
+![Manage civiform.dev dns records](../../../.gitbook/assets/dns-manage-custom-records.png)
+9. Scroll to the bottom of this list and click "Create new record" (note: the UI is a little buggy you might have to click this twice to get an empty row to appear).
+![Create new dns record](../../../.gitbook/assets/dns-create-new-record.png)
+
+10. Input the following details into the new row and click "Save":
+    - Host name: Input the value of the "CNAME name" column you copied in step 7 above. Remove the `civiform.dev` part at the end of the string. Google Domains automatically appends this to whatever you enter. If you do not remove it, you will add a CNAME record for `<random_id>.<dev-account>.civiform.dev.civiform.dev` which is incorrect.
+    - Type: Select "CNAME"
+    - TTL: Leave the default value "3600"
+    - Data: Input the value of the "CNAME value" column you copied in step 7 above.
+![Save new record details](../../../.gitbook/assets/dns-new-record-details.png)
+
+11. Navigate to the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) in AWS. It will take a minute or two for AWS to see and validate the new CNAME record. Once the validation has succeeded, the "Status" column of the "Certificates" list should have a green checkmark and say "Issued".
+![Certificate successfully issued](../../../.gitbook/assets/aws-cert-issued.png)
+
+## Setup AWS CLI
+
+1.  Create the credentials file
+    - Run `touch ~/.aws/credentials` in your terminal to create a config file for `aws-cli`.
```

Comment:
Perhaps `mkdir -p ~/.aws; touch ~/.aws/credentials` in case `~/.aws` doesn't exist?  Apparently the installer doesn't create this dir, on MacOS at least.